### ISSUE TITLE
2x4 Mixer Aux

### ISSUE THREAD

**Issue by Felnyrii** [mschack/VCV-Rack-Plugins#13]
I was loading up the mixer and noticed that when I clicked the Pre buttons for channels 1.3 & 1.4, it would activate for 2.1 & 2.2. Not detrimental to what I was currently doing, but seemed to be a malfunction.

![mschack_2x4_aux](https://user-images.githubusercontent.com/33522416/32977114-8a26f6de-cbec-11e7-9cf7-b61f5961fd0a.png)

Additionally, as a feature request, would you consider adding an aux return within the mixers? I understand I can use an addition mixer channel to bring the send back into the mix, but it just seemed cumbersome.

PS: Had some fun with the Synth Drums plugin on that patch, thanks!

**Comment by mschack** [comment]
Hi, thanks for the feedback! I have mainly focused on fixing and updating the four group mixer.  I will probably spend some time now updating the 1 and 2 group versions with those updates.  So to be clear you would like return inputs under each aux send output and put the return audio directly back into the mix?

**Comment by mschack** [comment]
Also I am glad you like the Synth Drums, it was a bit of an experiment to see what I could do.  I would like to work on it some more to see if I can expand the range of sounds you can make.

**Comment by Felnyrii** [comment]
Yeah, a stereo return that was just at unity gain would probably suffice. No need to make the return more cluttering to the layout.

I will have to try the 4x4 then, this is the first patch I have fully explored more than a track or two on.

**Comment by mschack** [comment]
I updated the 1 and 2 group mixers from the 4x4 code so hopefully that fixes the problem.

**Comment by Felnyrii** [comment]
2x Synth Drums
2x 3x16 Sequencers
1x 4x4 mixer 

[https://www.instagram.com/p/BbpJL8mlK4x/?taken-by=felnyrii](url)

**Comment by mschack** [comment]
Very cool!
